src/pages/[platform]/build-a-backend/add-aws-services/geo/set-up-geo/index.mdx

```diff
@@ -340,12 +337,23 @@ Initialized Amplify
 </InlineFilter>
 
 <InlineFilter filters={['swift']}>
-To initialize the Amplify Geo, use the `Amplify.addPlugin()` method to add the AWSLocationGeoPlugin. Next, finish configuring Amplify by calling `Amplify.configure()`.
+<Callout warning>
+Make sure to generate the `amplify_outputs.json` file by running the following command: 
+
+```bash title="Terminal" showLineNumbers={false}
+npx ampx sandbox
+```
```

Comment:
should this be moved outside the swift block? I think we should keep the prose relatively the same across platforms (with a few exceptions) for maintainability and a concise UX

Reply:
We could explore doing that in the future as a docs clean up, but right now each platform is doing its own thing

Reply:
platform-specific callouts were (mostly) cleaned up when content was ported into this new structure. swift has the callout to move the file into the project, which is fine, but prose should be relatively the same